index.html

```diff
@@ -0,0 +1,36 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Library Project JS</title>
+</head>
+
+<body>
+  <header>
+    <h1>Library Collaborative Project (JS)</h1>
+    <button class="btn-add-new-book">Add New Book</button>
+    <div class="new-book">
+      <form>
+        <label for="title">Book Title:</label><br>
+        <input type="text" id="book-title" name="book-title"><br>
+        <label for="author">Book Author:</label><br>
+        <input type="text" id="book-author" name="book-author"><br>
+        <label for="pages">Number of pages:</label><br>
+        <input type="text" id="pages-number" name="pages-number"><br>
+        <input type="checkbox" id="read-status" name="read-status" value="Book">
```

Comment:
- [ ] it would be better if the number of pages `<input>` is a `type` of `number`, not `text`, to reduce errors.